developer-docs-site/docs/guides/resource-accounts/key-rotation.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Key Rotation"
+id: "key-rotation"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Aptos Move accounts have authentication keys that are separate from their public address. This means you can rotate an account's private key to give control of it to another account without changing the initial account's public address.
+
+After rotating the key, the controlling account can sign transactions for the initial account.
+
+In this guide, we show examples for how to rotate an account's authentication key using the various Aptos SDKs.
```

Comment:
This mention various Aptos SDK, are we planning to add it to Rust SDK as well? Maybe we can remove line 20 for now, and follow up with Rust SDK later on when needed :)